WeightProducer/python/MCSampleValues.py

```diff
@@ -251,744 +251,497 @@ class MCSampleValuesHelper():
         },
         "TTTo2L2Nu" : {
             "CrossSection" : XSValues(
-                XS_13TeV=88.29, XSSource_13TeV="XSDB (NNLO)",
+                XS_13TeV=87.34, XSSource_13TeV="PDG XS - https://pdg.lbl.gov/2021/reviews/rpp2020-rev-top-quark.pdf",
```

Comment:
Did you multiply in by hand the branching fraction? If so, why not use the branching fraction mechanism built into the MCSample/MCSampleHelper class?

Reply:
Ah, implemented branching fraction mechanism like in other places 33f4d81